ext/native/base/stringutil.h

```diff
@@ -4,6 +4,7 @@
 #include <cstdio>
 #include <cstring>
 #include <string>
+#include <strings.h>
```

Comment:
This one is lower down, again now on Windows for cross plat reasons.

-[Unknown]